components/phpunit_bridge.rst

```diff
@@ -293,6 +301,76 @@ Running the following command will display the full stack trace:
 
     $ SYMFONY_DEPRECATIONS_HELPER='/Doctrine\\Common\\ClassLoader is deprecated\./' ./vendor/bin/simple-phpunit
 
+Testing with multiple version of PHPUnit
+----------------------------------------
+
+Use Case
+~~~~~~~~
+
+When testing a library that have to be compatible with serveral version of PHP
+at the same time (like Symfony does), because of dependencies, the test suite
+have to be tested by differentes version of PHPUnit. Unfortunatly, writing a
```

Comment:
... the test suite cannot use the latest versions of PHPUnit.

Reply:
I'd suggest removing the sentence that starts with `Unfortunatly` - we don't need to make a judgment here.